packages/server-wallet/server-wallet.api.md

```diff
@@ -225,19 +242,126 @@ export type SingleChannelOutput = {
+    // Warning: (ae-forgotten-export) The symbol "HoldingUpdatedArg" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    holdingUpdated({ channelId, amount, assetHolderAddress }: HoldingUpdatedArg): Promise<void>;
+    // (undocumented)
+    joinChannel({ channelId }: JoinChannelParams): Promise<SingleChannelOutput>;
+    // (undocumented)
+    joinChannels(channelIds: ChannelId[]): Promise<MultipleChannelOutput>;
+    // (undocumented)
+    knex: Knex;
+    // Warning: (ae-forgotten-export) The symbol "LedgerManager" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    ledgerManager: LedgerManager;
+    // (undocumented)
+    logger: Logger;
+    // Warning: (ae-forgotten-export) The symbol "Output" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    mergeMessages(output: Output[]): MultipleChannelOutput;
+    // Warning: (ae-forgotten-export) The symbol "ObjectiveManager" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    objectiveManager: ObjectiveManager;
+    // (undocumented)
+    pushMessage(rawPayload: unknown): Promise<MultipleChannelOutput>;
+    // (undocumented)
+    pushUpdate(rawPayload: unknown): Promise<SingleChannelOutput>;
+    // (undocumented)
+    registerAppBytecode(appDefinition: string, bytecode: string): Promise<void>;
+    // (undocumented)
+    registerAppDefinition(appDefinition: string): Promise<void>;
+    // Warning: (ae-forgotten-export) The symbol "Store" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    store: Store;
+    // (undocumented)
+    syncChannel({ channelId }: SyncChannelParams): Promise<SingleChannelOutput>;
+    // Warning: (ae-forgotten-export) The symbol "Bytes32" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    syncChannels(channelIds: Bytes32[]): Promise<MultipleChannelOutput>;
+    // (undocumented)
+    updateChannel({ channelId, allocations, appData, }: UpdateChannelParams): Promise<SingleChannelOutput>;
+    // (undocumented)
+    updateChannelFunding(args: UpdateChannelFundingParams): Promise<SingleChannelOutput>;
+    // Warning: (ae-forgotten-export) The symbol "UpdateChannelFundingParams" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    updateFundingForChannels(args: UpdateChannelFundingParams[]): Promise<MultipleChannelOutput>;
+    // (undocumented)
+    readonly walletConfig: ServerWalletConfig;
+    // (undocumented)
+    warmUpThreads(): Promise<void>;
+}
+
 // @public (undocumented)
 export function validateServerWalletConfig(config: Record<string, any>): {
     valid: boolean;
     value: ServerWalletConfig | undefined;
     errors: ValidationErrorItem[];
 };
 
-// Warning: (ae-forgotten-export) The symbol "SingleThreadedWallet" needs to be exported by the entry point index.d.ts
-//
```

Comment:
This is the warning that is silenced now. Of course, more warnings are now generated, but we are moving in the right direction.